cli/server/server.go

```diff
@@ -351,7 +370,15 @@ func startServer(ctx *cli.Context) error {
 	if err != nil {
 		return cli.NewExitError(fmt.Errorf("failed to create network server: %w", err), 1)
 	}
-	rpcServer := server.New(chain, cfg.ApplicationConfiguration.RPC, serv, log)
+	orc.SetOnTransaction(func(tx *transaction.Transaction) {
+		r := serv.RelayTxn(tx)
+		if r != network.RelaySucceed {
```

Comment:
I don't think logging is enough in this case, it's a bad error and if it happens, the module needs to react (at least by retrying).

Reply:
I am not sure if tx resend is responsibility of oracle module. If tx failed to verify (shouldn't happen at all), there is nothing we can do.